packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/VSphereCredentialsEdit.tsx

```diff
@@ -210,16 +210,12 @@ export const VSphereCredentialsEdit: React.FC<EditComponentProps> = ({ secret, o
         helperTextInvalid={state.validation.cacert.msg}
         validated={state.validation.cacert.type}
       >
-        <FileUpload
+        <CertificateUpload
```

Comment:
I understand why you dropped the `type="text"` and `onDataChange={(value) => handleChange('cacert', value)}` because certs are usually texts.

I have concerns with the props changes:

a. changing the props means it's not a drop in replacement
b. removing `filenamePlaceholder` and `browseButtonText`, remove the options to change text lables or use translation when needed ( t can only be used from inside the app ).
c. removing `onClearClick` remove the option to control component behaviour
d. renaming `onTextChange` to  `handleSave` changes the way we interact with the component

Reply:
OK. I understand correctly you would like to keep exactly the same interface i.e.
```
export interface CertificateUploadProps  extends FileUploadProps {
```